src/main/resources/swagger/api-docs.yaml

```diff
@@ -3644,6 +3644,49 @@ paths:
           description: Internal Server Error
           content: {}
       x-passthrough: false
+  /api/workspaces/{workspaceNamespace}/{workspaceName}/entities/{entityType}/tsv/save:
+    post:
```

Comment:
I don't like this URL signature … any suggestions for something better? The existing download-TSV API is `GET /api/workspaces/{workspaceNamespace}/{workspaceName}/entities/{entityType}/tsv`; I think the download-to-bucket API should be similar to that, but "…/tsv/save" feels hacky.